content/docs/how-tos/onboarding-a-new-component.md

```diff
@@ -0,0 +1,295 @@
+are delivered from a repo or org. Today, we have organization-level webhooks delivered for all repos in the following
+orgs:
+
+* openshift
+* openshift-s2i
+* operator-framework
+
+
+If you are onboarding a new component in one of these orgs, you do not need to configure webhooks individually for your
+repository, please continue on to the next section.
+
+#### Token Retrieval
+
+Webhook deliveries to the CI system are encrypted with an HMAC token provided when configuring the webhook in the GitHub
+UI. A CI administrator can configure the webhook appropriately for your repository if you grant them edit access, or you
+can contact a CI administrator to get a copy of the token so that you may add the webhook yourself.
+
+Contact a CI administrator via the @dptp-helpdesk alias in the #forum-testplatform channel in the CoreOS slack.
+
+#### Webhook Setup
+
+In order to configure the webhook delivery, you will need edit access to the repository you are onboarding. To set up a
+webhook, navigate through the GitHub UI to the settings page for your repository, then choose “Webhooks” from the menu
+on the left. Your webhook should be set up with the following parameters:
+
+|Parameter|Value|
+|:---|:---|
+|Payload URL|https://hook.ci.openshift.org/hook|
+|Content Type|`application/json`|
+|Secret|see [Token Retrieval](#Token Retrieval) above|
+|SSL Verification|enabled|
+|Event types|all|
+|Active|yes|
+
+*Note:*  Note: Don’t add `/` at the end of the payload url.
+
+### Granting Robots Privileges
+
+In order to add labels, move PRs and issues into milestones, merge PRs, etc, the robots will need write access to your repository.
+
+If your repository is in the OpenShift organization and it was created by Dev Productivity team, it probably has Team
+OpenShift Robots added by default as part of the repo creation process. To check if your repo has the robots added,
+please ask in slack in #forum-dp-platform or open a ticket. If your repo does not have robots, then open a new ticket
+in their [JIRA project](https://jira.coreos.com/projects/DPP/summary) with the details.
+
+If your component repository is **not** in this organization:
+
+1. Invite `openshift-ci-robot` and `openshift-merge-robot` into your organization or add them as collaborators for the repository.
+1. Contact a CI administrator to accept the invitation
+1. Contact a CI administrator to accept the invitation
+1. By default, we enabled [branch-protection](TODO: Add link once https://github.com/openshift/ci-docs/pull/7 merges) for
+	 all prow-controlled repos. This requires `openshift-merge-robot` to be an admin of the repo. We can disable it in
+   prow’s [config.yaml](https://github.com/openshift/release/blob/master/core-services/prow/02_config/_config.yaml)
+
+
+Once you have granted the requisite privileges, you will need to enable Prow plugins before testing. If you enable the
```

Comment:
Might be outdated, should likely tell them to run `make new-repo` in `openshfit/release` and worry about this later, not in this section.